cpp/modmesh/view/RPythonConsoleDockWidget.hpp

```diff
@@ -115,14 +115,9 @@ public slots:
     std::deque<std::string> m_past_command_strings;
     int m_current_command_index = 0;
     size_t m_past_limit = 1024;
+    int m_last_command_serial = 0;
```

Comment:
`m_last_command_serial` is for showing in the console text area.  `m_current_command_index` is the index to `m_past_command_strings`.  It is possible that `m_last_command_serial > m_past_command_strings.size()`.